pom.xml

```diff
@@ -1,334 +1,254 @@
 <?xml version="1.0" encoding="UTF-8"?><project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
+  <modelVersion>4.0.0</modelVersion>
 
-    <groupId>projects</groupId>
-    <artifactId>mp-istio-sample-servicea</artifactId>
-    <version>1.0-SNAPSHOT</version>
-    <packaging>war</packaging>
+  <groupId>projects</groupId>
+  <artifactId>service-a</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <packaging>war</packaging>
 
-    <!-- contains default pluginManagement configuration for liberty-maven-plugin -->
-    <parent>
-        <groupId>net.wasdev.wlp.maven.parent</groupId>
-        <artifactId>liberty-maven-app-parent</artifactId>
-        <version>2.2</version>
-    </parent>
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
 
-    <properties>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
+    <app.name>service-a</app.name>
+    <testServerHttpPort>9080</testServerHttpPort>
+    <testServerHttpsPort>9443</testServerHttpsPort>
+    <warContext>${app.name}</warContext>
+    <package.file>${project.build.directory}/${project.artifactId}-${project.version}.zip</package.file>
+    <packaging.type>usr</packaging.type>
 
-        <app.name>mp-istio-sample-a</app.name>
-        <testServerHttpPort>9080</testServerHttpPort>
-        <testServerHttpsPort>9443</testServerHttpsPort>
-        <warContext>${app.name}</warContext>
-        <package.file>${project.build.directory}/${project.artifactId}-${project.version}.zip</package.file>
-        <packaging.type>usr</packaging.type>
+    <!-- Plugins -->
+    <version.maven-war-plugin>2.6</version.maven-war-plugin>
+    <version.liberty-maven-plugin>2.2</version.liberty-maven-plugin>
```

Comment:
This should be a property in the Liberty profile, along with other Liberty profile specific plugins, etc